projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -60,6 +75,39 @@ const Menu = ( { items = [], onManage } ) => {
 					>
 						{ __( 'Manage', 'jetpack-my-jetpack' ) }
 					</Button>
+					{ showStandaloneOption && (
+						<>
+							<hr />
+							{ ! isStandaloneInstalled && (
+								<Button
+									weight="regular"
+									fullWidth
+									variant="tertiary"
+									icon={ arrowDown }
+									onClick={ () => {
+										onClose();
+										onInstall?.();
+									} }
+								>
+									{ __( 'Install plugin', 'jetpack-my-jetpack' ) }
+								</Button>
+							) }
+							{ isStandaloneInstalled && ! isStandaloneActive && (
+								<Button
+									weight="regular"
+									fullWidth
+									variant="tertiary"
+									icon={ external }
```

Comment:
I think we should use another icon, this one is same than `Manage`, maybe the `plugins` one or [another one](https://automattic.github.io/jetpack-storybook/?path=/story/gutenberg_icons-icon--library)